lib/cog_api/http/api_response.ex

```diff
@@ -39,6 +39,9 @@ defmodule CogApi.HTTP.ApiResponse do
   def format_delete(%Response{}, error_message) do
     {:error, [error_message]}
   end
+  def format_delete({:error, err}, error_message) do
+    {:error, [err <> ": " <> error_message]}
```

Comment:
Is there a more descriptive name for these variables? `server_error` and `client_error` maybe?